__tests__/nc_news.test.js

```diff
@@ -117,3 +118,55 @@ describe('GET /api/articles/:article_id', () => {
 		});
 	});
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+	describe('Endpoint Behaviour', () => {
+		test('GET:200 expects correct status code', () => {
+			return request(app).get('/api/articles/1/comments').expect(200);
+		});
+		test('GET:200 expects correct comment format & correct quantity', () => {
+			return request(app)
+				.get('/api/articles/1/comments')
+				.then(({ body }) => {
+					const comments = body.comments;
+					expect(comments).toHaveLength(11);
+					comments.forEach((comment) => {
+						expect(typeof comment.comment_id).toBe('number');
+						expect(typeof comment.votes).toBe('number');
+						expect(typeof comment.created_at).toBe('string');
+						expect(typeof comment.author).toBe('string');
+						expect(typeof comment.body).toBe('string');
+						expect(typeof comment.article_id).toBe('number');
+					});
+				});
+		});
+		test('GET:200 expects correct ordered by default from newest ASC', () => {
+			return request(app)
+				.get('/api/articles/1/comments')
+				.then(({ body }) => {
+					const comments = body.comments;
+					expect(comments).toBeSortedBy('created_at', {
+						descending: false,
+					});
+				});
+		});
+	});
+	describe('Endpoint error handling', () => {
+		test('GET:404 expects error when ID does not exist', () => {
+			return request(app)
+				.get('/api/articles/600/comments')
+				.expect(404)
+				.then(({ body }) => {
+					expect(body.msg).toBe('Article ID does not exist');
+				});
+		});
+		test('GET:400 expects error when ID not a valid type', () => {
+			return request(app)
+				.get('/api/articles/cheese/comments')
+				.expect(400)
+				.then(({ body }) => {
+					expect(body.msg).toBe('Article ID must be a number');
+				});
+		});
+	});
```

Comment:
Please include a test here where the article exists, but there are no comments yet. This should respond with an empty array and status 200.